ale/drivers/mex_drivers.py

```diff
@@ -422,3 +495,85 @@ def sensor_model_version(self):
           ISIS sensor model version
         """
         return 1
+
+class MexHrscIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, RadialDistortion, Driver):
+
+        @property
+        def instrument_id(self):
+            if(super().instrument_id != "HRSC"):
+                raise Exception ("Instrument ID is wrong.")
+
+            return super().instrument_id
+
+        @property
+        def sensor_model_version(self):
+            return 1
+
+        @property
+        def line_scan_rate(self):
+            isis_bytes = read_table_data(self.label['Table'], self._file)
+            times_table = parse_table(self.label['Table'], isis_bytes)
+            start_time = times_table['EphemerisTime'][0]
+            return times_table['LineStart'], times_table['EphemerisTime'], times_table['ExposureTime']
+
+        @property
+        def ephemeris_start_time(self):
+            isis_bytes = read_table_data(self.label['Table'], self._file)
+            times_table = parse_table(self.label['Table'], isis_bytes)
+            start_time = times_table['EphemerisTime'][0]
+            return start_time
+
+        @property
+        def ephemeris_stop_time(self):
+            isis_bytes = read_table_data(self.label['Table'], self._file)
+            times_table = parse_table(self.label['Table'], isis_bytes)
+            last_line = times_table['LineStart'][-1]
+            stop_time = times_table['EphemerisTime'][-1] + ((self.image_lines - last_line + 1) * times_table['ExposureTime'][-1])
+            return stop_time
+
+        @property
+        def ikid(self):
+            """
+            Returns the Naif ID code for the HRSC head instrument
+
+            This would be the Naif ID code for the base (or "head") instrument.
+
+            Returns
+            -------
+            : int
+              Naif ID used to for indentifying the instrument in Spice kernels
+            """
+            return spice.bods2c("MEX_HRSC_HEAD")
+
+        @property
+        def center_ephemeris_time(self):
+            """
+            Returns the center ephemeris time.
+
+            For HRSC, the center ephemeris time is calculated with the ephemeris stop time,
+            which is calculated from the binary image data.
+
+            Returns
+            -------
+            : float
+              Center ephemeris time
+            """
+            return (self.ephemeris_stop_time + self.ephemeris_start_time) / 2
```

Comment:
This is already implemented in base https://github.com/USGS-Astrogeology/ale/blob/master/ale/base/base.py#L308